### ISSUE TITLE
Map Screenshot Feedback - Phantom Forest / Crimsonwood

### ISSUE THREAD

**Issue by PirateIzzy** [Elem8100/MapleStory-GM-Client#12]
Hello, I was wondering if it was possible to implement a couple things in regard to saving maps:

1. Disabling Map Front effects.
2. Map Background alignment.

I saved a map in the GM Client for reference:
![image](https://user-images.githubusercontent.com/34289651/85957926-873a5480-b95f-11ea-8972-dc87300a0c4c.png)

As you can see in regards to 1, the sanity effect is displayed on the bottom left. It is disabled when you uncheck the "Back" Display option, but it also disables the map background. WzComparerR2 has an option to disable the Map Front, which hides the sanity effect while leaving the background alone.

In regards to 2, the map background is not aligned properly, and looks awkward on the top half (admittedly an issue with WzComparerR2 as well).

**Comment by Elem8100** [comment]
1-> Done
2-> You can change "Returen Y Position" value  when save map
download  v3.33
![snap](https://user-images.githubusercontent.com/49195287/86002309-c1305700-ba42-11ea-965f-65f3f050fab1.jpg)

**Comment by PirateIzzy** [comment]
1 works now, thanks.

As for 2, I tried with multiple Y position / Ratio alignments, but haven't been able to get a clean image of that map.

Tried it with Y Position -500, Ratio -2:
![610020002-Lower Ascent Y-500 R-2](https://user-images.githubusercontent.com/34289651/86078129-6a834700-ba5b-11ea-8881-ce29fe76f8e3.png)

Y Position -1000, Ratio -2:
![610020002-Lower Ascent Y-1000 R-2](https://user-images.githubusercontent.com/34289651/86078149-7a029000-ba5b-11ea-90de-7c74712927c9.png)

Y Position +200, Ratio +1:
![610020002-Lower Ascent Y+200 R+1](https://user-images.githubusercontent.com/34289651/86078166-838bf800-ba5b-11ea-94ed-4a4a6e6ebdbf.png)

Y Position +200, Ratio -2:
![610020002-Lower Ascent Y+200 R-2](https://user-images.githubusercontent.com/34289651/86078193-8f77ba00-ba5b-11ea-8078-813f26b24d60.png)

Is there any resolution to this, or is it a limitation of this specific map (being too tall of a map)?

**Comment by Elem8100** [comment]
I don't quite understand what you mean. Aligned bottom? or aligned top ?
Some images is not background they are "Obj", the position is depend on wz data.There is no way to change that.
![top](https://user-images.githubusercontent.com/49195287/86092931-c9d76a80-bae0-11ea-8066-886199a48c4c.png)

![610020002-Lower Ascent](https://user-images.githubusercontent.com/49195287/86093001-e2e01b80-bae0-11ea-8c3f-dd36c6a8f9b1.png)

**Comment by PirateIzzy** [comment]
I was curious if the background could be adjusted so that it doesn't have a large empty red space, but it might not be possible given how large the map is. Oh well, thank you anyway.

**Comment by PirateIzzy** [comment]
So I came across a map rendering of the above map (Lower Ascent, ID 610020002) on Namu: 
https://namu.wiki/w/%EB%A7%88%EC%8A%A4%ED%85%8C%EB%A6%AC%EC%95%84(GMS)

This is the map in question, with the background covering the entire map.
![6af6272103ac22ca73a9a0f7b752c6b97810384ed74f73efcdf3bc69cf8a71a748b68d909cb74bcb611df7205c6766e98e5b](https://user-images.githubusercontent.com/34289651/92325957-97844500-f01c-11ea-9be8-5c63bcf699ed.png)

I was wondering if it was possible to use the GM Client map screenshot / renderer to make it look like the above. When I tried it, this was the result, looking distorted:
![20200903155934856_610020002](https://user-images.githubusercontent.com/34289651/92325956-8f2c0a00-f01c-11ea-8589-cc109b23c331.png)

Would it be feasible to allow for a higher negative ratio, i.e. beyond -2 (maybe up to -10)?
